tests/unit/r2i/tflite/Makefile.am

```diff
@@ -36,22 +36,22 @@ LOG_DRIVER=$(top_builddir)/common/rr-test-driver.py $(SINGLE_PROCESS)
 AM_CXXFLAGS =                       \
         $(RR_CXXFLAGS)              \
         $(TESTS_CFLAGS)             \
-        $(TFLITE_CFLAGS)    \
+        $(TFLITE_CFLAGS)            \
         $(CODE_COVERAGE_CXXFLAGS)
 
 AM_CFLAGS =                         \
         $(RR_CFLAGS)                \
         $(TESTS_CFLAGS)             \
-        $(TFLITE_CFLAGS)    \
+        $(TFLITE_CFLAGS)            \
         $(CODE_COVERAGE_CFLAGS)
 
 AM_CPPFLAGS =                       \
         $(RR_CPPFLAGS)              \
         $(CODE_COVERAGE_CPPFLAGS)
 
-AM_LDFLAGS =                        \
+LDADD =                             \
         $(RR_LIBS)                  \
-        $(TFLITE_LIBS)      \
+        $(TFLITE_LIBS)              \
```

Comment:
This should no longer be necessary. Libtool should handle the inter-library dependencies.